openapi/components/schemas/ThreeDSecureServers/EMerchantPay3dsServers/eMerchantPay3dsServer.yaml

```
@@ -1,4 +1,10 @@
+type: object
 description: eMerchantPay 3DS server.
-allOf:
-  - $ref: ./eMerchantPay3dsServers.yaml
-
+required:
+  - name
+properties:
+  name:
+    type: string
+    description: Merchant plug-in name.
```

suggestion: suggestion for readability (series of nouns)
```suggestion
    description: Name of the merchant plug-in.
```